tincr/io/design/rapidsmith.tcl

```diff
@@ -0,0 +1,238 @@
+    }
+    
+    puts -nonewline $returnString
+}
+
+## Prints the wires, cell pins, and bel pins of the net to standard out in the following format: <br>
+#
+#   "wire1 wire2 ... \n" <br>
+#   "pin1 pin2 ... pin3" <br>
+#   "belpin1 belpin2 ... belpin3" <br>
+#
+#   This function is used to verify that a net has been imported correctly into RapidSmith
+#
+proc ::tincr::get_physical_net_info {net} {
+    set_param tcl.collectionResultDisplayLimit 100000
+    
+    set net [get_nets $net]
+    
+    set wires [get_wires -of $net -quiet]
+    set cell_pins [get_pins -of $net -quiet]
+    set bel_pins [create_bel_pin_string $net]
+    
+    puts "$wires\n$cell_pins\n$bel_pins"
+}
+
+## Scans through the attached bel pins of the specified net, and replaces any IOB site
+#   names with the PACKAGE_PIN name of the site. This is to match RapidSmith.
+#
+# @param Name of a net
+# @return a list of bel pins connected to the net with site pin names adjusted.
+proc create_bel_pin_string {net} {
+    set bel_pin_string ""
+    foreach bel_pin [get_bel_pins -of $net -quiet] {
+        set bel_pin_toks [split $bel_pin "/"]
+        set site [get_sites [lindex $bel_pin_toks 0]]
+        
+        if { [get_property IS_PAD $site] } {
+            set site [get_property NAME [get_package_pins -quiet -of_object $site]]
+        }
+        
+        set bel_pin_string "$bel_pin_string$site/[lindex $bel_pin_toks 1]/[lindex $bel_pin_toks 2] "
+    }
+    return $bel_pin_string
+}
+
+## Prints the wires, cell pins, and bel pins of ALL VCC nets in the currently opened design
+#   The output format is the same as {@link ::tincr::get_physical_net_info}
+proc ::tincr::get_vcc_routing_info {} {
+    
+    set_param tcl.collectionResultDisplayLimit 100000
+    
+    set vcc_nets [get_nets -filter {TYPE==POWER}]
+    
+    set num_vcc_cells [llength [get_cells -filter {REF_NAME==VCC}]]
+    set vcc_wires [get_wires -of $vcc_nets -quiet]
+    set vcc_cell_pins [get_pins -of $vcc_nets -quiet]
+    set vcc_bel_pins [get_bel_pins -of $vcc_nets -quiet]
+        
+    puts "$vcc_wires\n$num_vcc_cells\n$vcc_cell_pins\n$vcc_bel_pins"
+}
+
+## Prints the wires, cell pins, and bel pins of ALL GND nets in the currently opened design
+#   The output format is the same as {@link ::tincr::get_physical_net_info}
+proc ::tincr::get_gnd_routing_info {} {
+    
+    set_param tcl.collectionResultDisplayLimit 100000
+    
+    set num_gnd_cells [llength [get_cells -filter {REF_NAME==GND}]]
+    set gnd_nets [get_nets -filter {TYPE==GROUND}]
+    set gnd_wires [get_wires -of $gnd_nets -quiet]
+    set gnd_cell_pins [get_pins -of $gnd_nets -quiet]
+    set gnd_bel_pins [get_bel_pins -of $gnd_nets -quiet]
+    
+    puts "$gnd_wires\n$num_gnd_cells\n$gnd_cell_pins\n$gnd_bel_pins"
+}
+
+## Returns the number of used sites in the currently opened design
+#
+proc ::tincr::get_used_site_count {} {
```

Comment:
Rename to `report_used_site_count` since you're just printing information to stdout and not returning anything.